src/main.ts

```diff
@@ -0,0 +1,78 @@
+import {getInput, saveState, setFailed} from '@actions/core'
+import {exec} from '@actions/exec'
+import {info} from 'console'
+import optionMappingJson from './option-mapping.json'
+
+const CONTAINER_VERSION = '4.6.2'
+const LOG_FILE = '/srv/sauce-connect.log'
```

Comment:
Is it possible to upload this log file as asset?

Reply:
possible, refs https://github.com/actions/toolkit/tree/main/packages/artifact

Is it OK to make this change in this PR? I assume that we can handle this new feature in another PR.

Reply:
> I assume that we can handle this new feature in another PR.

Feel free to add it as part of this PR. If not, let's create an issue for it an tackle it afterwards.